core/src/main/resources/hudson/model/AbstractBuild/index.jelly

```diff
@@ -29,34 +29,26 @@ THE SOFTWARE.
     <st:include page="sidepanel.jelly" />
     <!-- no need for additional breadcrumb here as we're on an index page already including breadcrumb -->
     <l:main-panel>
-      <div style="float:right; z-index: 1; position:relative; margin-left: 1em">
+      <div style="float: right; z-index: 1; position: relative; margin-left: 1em">
         <l:hasPermission permission="${it.UPDATE}">
           <st:include page="logKeep.jelly" />
         </l:hasPermission>
-        <div style="margin-top:1em">
-          ${%startedAgo(it.timestampString)}
-          </div><div>
-          <j:if test="${it.building}">
-            ${%beingExecuted(it.executor.timestampString)}
-          </j:if>
-          <j:if test="${!it.building}">
-            ${%Took} <a href="${rootURL}/${it.parent.url}buildTimeTrend">${it.durationString}</a>
-          </j:if>
-          <j:if test="${!empty(app.nodes) or (it.builtOnStr != null and it.builtOnStr != '')}">
-            ${%on} <t:node value="${it.builtOn}" valueStr="${it.builtOnStr}"/>
-          </j:if>
+        <div style="margin-top: 1em"> ${%startedAgo(it.timestampString)} </div
+        ><div>
+          <j:if test="${it.building}"> ${%beingExecuted(it.executor.timestampString)} </j:if>
+          <j:if test="${!it.building}"> ${%Took} <a href="${rootURL}/${it.parent.url}buildTimeTrend">${it.durationString}</a> </j:if>
+          <j:if test="${!empty(app.nodes) or (it.builtOnStr != null and it.builtOnStr != '')}"> ${%on} <t:node value="${it.builtOn}" valueStr="${it.builtOnStr}" /> </j:if>
```

Comment:
Does this look like a net positive change to anyone? Much longer lines, pointless wrapping of content in spaces, weird
```
</div
>
```
line break, …

Reply:
```
 [INFO] Running 'yarn lint:ci' in /home/jenkins/agent/workspace/Core_jenkins_PR-8766/war
 [INFO] Checking formatting...
 [INFO] [warn] ../core/src/main/resources/hudson/rss20.jelly
 [INFO] [warn] ../core/src/main/resources/lib/hudson/progressBar.jelly
 [INFO] [warn] Code style issues found in 2 files. Run Prettier to fix.